src/features/organizations/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface TreeItemData {
+  id: number;
+  title: string;
+  children: TreeItemData[] | [];
+}
```

Comment:
I put this in a separate module because it's used both on the very backend (RPC) and on the very frontend (inside components). I don't feel comfortable importing stuff from an RPC to a component.